packages/task/src/browser/task-service.ts

```diff
@@ -193,6 +194,9 @@ export class TaskService implements TaskConfigurationClient {
     @inject(TaskTerminalWidgetManager)
     protected readonly taskTerminalWidgetManager: TaskTerminalWidgetManager;
 
+    @inject(TaskContextKeyService)
+    protected readonly taskContextKeyService: TaskContextKeyService;
```

Comment:
I suspect this is to force the creation of an instance?

Reply:
Yes, we don't really use those currently, and otherwise the service does not get created :/